packages/lab/src/__tests__/__e2e__/slider/RangeSlider.cy.tsx

```diff
@@ -330,6 +397,46 @@ describe("Given a Range Slider", () => {
     cy.findAllByTestId("sliderTooltip").eq(1).should("have.text", "4%");
   });
 
+  it("should round to the decimal places provided", () => {
+    cy.mount(
+      <Default
+        min={0}
+        max={4.3}
+        step={0.375}
+        decimalPlaces={2}
+        defaultValue={[0, 0.375]}
+      />,
+    );
+
+    cy.findAllByRole("slider").eq(1).focus().realPress("ArrowRight");
+    cy.findAllByRole("slider").eq(1).should("have.attr", "value", 0.75);
+    cy.findAllByRole("slider").eq(1).focus().realPress("ArrowRight");
+    cy.findAllByRole("slider").eq(1).should("have.attr", "value", 1.13);
+  });
+
+  it("should restrict labels from overflowing slider boundaries when set", () => {
```

Comment:
is this better off as a visual QA test, testing the class exists is no guarantee that it's not broken